cfgov/jinja2/v1/_includes/organisms/expandable.html

```diff
@@ -33,62 +33,52 @@
 
 {% macro expandable(value, expandable_cue_additional_text='') %}
 
-<div data-qa-hook="expandable" class="o-expandable
-            o-expandable__expanded
-            {{ 'o-expandable__borders' if value.is_bordered else '' }}
-            {{ 'o-expandable__midtone' if value.is_midtone else '' }}"
-            {{ 'data-state=expanded' if value.is_expanded else '' }}>
-
-    <button type="button" class="o-expandable_target">
-        <div class="o-expandable_header">
-            <span class="o-expandable_header-left
-                         o-expandable_label">
-                {{ value.label }}
-            </span>
-            <span class="o-expandable_header-right
-                         o-expandable_link
-                         u-hidden">
-                <span class="o-expandable_cue
-                             o-expandable_cue-open">
-                    <span class="o-expandable_cue-label
-                                {{ 'u-hidden' if value.hide_cue_label else '' }}">
-                                Show {{expandable_cue_additional_text}}</span>
-                    <span class="o-expandable_cue-label
-                                 o-expandable_cue-label__es
-                                {{ 'u-hidden' if value.hide_cue_label else '' }}">
-                                Mostrar</span>
-                    {{ svg_icon('plus-round') }}
+<div data-qa-hook="expandable"
```

Comment:
Hmm I wonder if this is used.

Reply:
Not sure, but you'll probably want to follow up.